src/name_bazaar/ui/events/ens_events.cljs

```diff
@@ -103,3 +117,39 @@
   (fn [{:keys [:db]} [node]]
     (let [{:keys [:ens.record/owner]} (get-ens-record db node)]
       {:dispatch [:public-resolver.name/load owner]})))
+
+(reg-event-fx
+  :ens.records/setup-public-resolver
+  [interceptors (validate-first-arg (s/keys :req [:ens.record/name]))]
+  (fn [{:keys [:db]} [form-data]]
+    (let [public-resolver (get-in db [:smart-contracts :public-resolver :address])
+          form-data (assoc form-data
+                           :ens.record/node (namehash (str (:ens.record/name form-data)
+                                                           constants/registrar-root))
+                           :public-resolver public-resolver)]
+      (info [:SET-RESOLVER-NODES name public-resolver])
+      {:dispatch [:district0x/make-transaction
+                  {:name (gstring/format "Setting resolver for %s" (:ens.record/name form-data))
+                   :contract-key :ens
+                   :contract-method :set-resolver
+                   :form-data (select-keys form-data [:ens.record/node :public-resolver])
+                   :args-order [:ens.record/node :public-resolver]
+                   :form-id (select-keys form-data [:ens.record/node])
+                   :tx-opts {:gas 100000 :gas-price default-gas-price}
```

Comment:
About `:result-href` here: Into `/manage-names` page we should add query parameter like `/manage-names?name=johnydoe.eth` that will prefill both "Point..." forms with that name. So users can easily navigate back after resolver is set up. Such url will be `:result-href` here. This is already implemented for create offering form like `/create-offering?name=`, so you can take a look there.